File: parent327/pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillQueryService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillOrder;
import entity.PageResult;

public interface SeckillQueryService {
    PageResult search(Integer pageNum, Integer pageSize, SeckillOrder seckillOrder);

    SeckillOrder findOne(Long id);
}

File: parent327/pinyougou-web-manager/src/main/webapp/js/service/seckillQueryService.js
// 定义服务层:
app.service("seckillQueryService",function($http){

    this.findOne=function(id){
        return $http.get("../seckillQuery/findOne.do?id="+id);
    }
    this.search = function(page,rows,searchEntity){
        return $http.post("../seckillQuery/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
    }
});